extensions/mssql/package.json

```diff
@@ -18,10 +18,15 @@
     "update-grammar": "node ../../build/npm/update-grammar.js Microsoft/vscode-mssql syntaxes/SQL.plist ./syntaxes/sql.tmLanguage.json"
   },
   "dependencies": {
-    "dataprotocol-client": "github:Microsoft/sqlops-dataprotocolclient#0.2.15",
+    "clipboardy": "^1.2.3",
```

Comment:
I don't see where this is being used.

Reply:
Yurong - are you sure we need this? Per Anthony's comment it'd be good to avoid adding this if not needed.

Reply:
When VS Code 1.30+ is merged, the `Clipboard` interface can be used instead.